packages/main/src/plugin/recommendations/recommendations-registry.ts

```diff
@@ -16,12 +16,56 @@
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
 
-import type { IConfigurationNode, IConfigurationRegistry } from '/@/plugin/configuration-registry.js';
+import type { ConfigurationRegistry, IConfigurationNode } from '/@/plugin/configuration-registry.js';
+import type { Featured } from '/@/plugin/featured/featured.js';
+import type { FeaturedExtension } from '/@/plugin/featured/featured-api.js';
+import type { ExtensionBanner } from '/@/plugin/recommendations/recommendations-api.js';
 
+import { default as recommendations } from '../../../../../recommendations.json';
 import { RecommendationsSettings } from './recommendations-settings.js';
 
 export class RecommendationsRegistry {
-  constructor(private configurationRegistry: IConfigurationRegistry) {}
+  constructor(
+    private configurationRegistry: ConfigurationRegistry,
+    private featured: Featured,
+  ) {}
+
+  isRecommendationDisabled(): boolean {
+    return this.configurationRegistry
+      .getConfiguration(RecommendationsSettings.SectionName)
+      .get<boolean>(RecommendationsSettings.IgnoreRecommendations, false);
+  }
```

Comment:
I think it works better for readability when the boolean is true if enabled and false if disabled

`isRecommandationsEnabled()`

Reply:
Why not, I added two tests to ensure expected behaviour with the renamed method.

Reply:
It's not about the tests,

it's that when something is disabled, then you have what we could called double negation like

'not (disabled)' meaning 'not not enabled' (meaning enabled)